src/events/index.js

```diff
@@ -160,6 +160,74 @@ class Events {
       .then((event) => toCamelCase(event));
   }
 
+  /**
+   * Gets all event types for a client
+   *
+   * API Endpoint: '/clients/:clientId/types'
+   * Method: GET
+   *
+   * @param {string} clientID The ID of the client
+   *
+   * @returns {Promise}
+   * @fulfill {Event} Information about an event
```

Comment:
Is this returning a single event or all event types related to a client? On line 164 it looks like it's returning all `EventTypes` related to a specific client but this reads like it's returning a single event?